src/Workspaces/Core/Portable/CodeRefactorings/FixAllOccurences/DocumentBasedFixAllProvider.cs

```diff
@@ -84,16 +84,13 @@ public sealed override IEnumerable<FixAllScope> GetSupportedFixAllScopes()
     /// them, and are used to perform a final cleanup pass for formatting/simplication/etc.  Text is returned for
     /// documents that don't support syntax.
     /// </summary>
-    private async Task<Dictionary<DocumentId, (SyntaxNode? node, SourceText? text)>> GetFixedDocumentsAsync(
-        FixAllContext fixAllContext, IProgress<CodeAnalysisProgress> progressTracker)
+    private async Task<Dictionary<DocumentId, (SyntaxNode? node, SourceText? text)>> GetFixedDocumentsAsync(FixAllContext fixAllContext)
     {
         Contract.ThrowIfFalse(fixAllContext.Scope is FixAllScope.Document or FixAllScope.Project
             or FixAllScope.ContainingMember or FixAllScope.ContainingType);
 
         var cancellationToken = fixAllContext.CancellationToken;
 
-        using var _1 = progressTracker.ItemCompletedScope();
```

Comment:
we definitely weren't doing 'two' "completed" calls here.  so this was entirely busted for progress.